alignak/objects/schedulingitem.py

```diff
@@ -1763,6 +1763,11 @@ def consume_result(self, chk, notif_period, hosts,  # pylint: disable=R0915,R091
             if self.state_type == 'SOFT':
                 if not chk.is_dependent():
                     self.add_attempt()
+                # Cases where go:
+                #  * warning soft => critical hard
```

Comment:
If we are in critical hard, then we are not in this case (see above, state_type == SOFT)